packages/custom-storage/src/StorageRepository.ts

```diff
@@ -0,0 +1,37 @@
+import { ColumConfig } from '@carto/toolkit-sql/dist/types/DDL';
+
+export interface StoredVisualization extends Visualization {
+  id: string;
+}
+
+export interface CompleteVisualization {
+  vis: StoredVisualization;
+  datasets: Dataset[];
+}
+
+export interface Visualization {
+  name: string;
+  desc: string;
+  img: string;
```

Comment:
why not a full 'description' and 'image' here?

PD. I wanted to use the new 'multi-line comments' for sure...  😛

Reply:
Good question, I don't know

Actually, `desc` is an issue with SQL, so initially it was also `desc` for the DDL, but I had to change it. Also, `img` is `thumbnail` on the DB, which is a big inconsistency.

Reply:
Let's try to keep them homogeneous then

Reply:
Already fixed